lib/iiif_manifest/v3/manifest_service_locator.rb

```diff
@@ -88,6 +89,14 @@ def choice_builder
           )
         end
 
+        def supplementing_content_builder
+          IIIFManifest::ManifestServiceLocator::InjectedFactory.new(
+            ManifestBuilder::SupplementingContentBuilder,
+            iiif_annotation_factory: iiif_annotation_factory,
+            body_builder_factory: body_builder_factory
```

Comment:
I think this needs to be the `supplementing_body_builder_factory` in order for the language property to be added.
```suggestion
            body_builder_factory: supplementing_body_builder_factory
```